apps/cowswap-frontend/src/common/updaters/orders/PendingOrdersUpdater.ts

```diff
@@ -200,7 +200,7 @@ async function _updateOrders({
 
   if (presigned.length > 0) {
     // Only mark as presigned the orders we were not aware of their new state
-    const presignedOrderIds = presigned as OrderID[]
+    const presignedOrderIds = presigned as string[]
```

Comment:
Is the casting still needed?

Reply:
<img width="1275" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/6419c6b1-9deb-4445-a612-90a7d4aba5f9">

Yep

Reply:
but I guess is because it can be one of these two types: `OrderFulfillmentData | string`

Why do we assume is a string? or maybe the question is, why did we do such a strange type?

Reply:
Sorry, I don't know, it also looks weird to me